src/style.css

```
@@ -1 +1,16 @@
-/* styles go here */
+fieldset {
+  margin-bottom: 20px;
+}
+
+label {
+  display: block;
+  margin-bottom: 10px;
+}
```

Tag selector is slower than class selector. so I suggest to make it like:
```suggestion
.fieldset {
  margin-bottom: 20px;
}

.input-label {
  display: block;
  margin-bottom: 10px;
}

...

<label for="recommend" class="input-label">
<fieldset class="fieldset">
```